### PR TITLE
[ko] Korean Translation for Lesson 6 

### PR DESCRIPTION
**Pull request by jhYun505**
### 6-1 : speech-recognition
- pi-audio.ko.md

### 6-2 : language-understanding
- README.ko.md
- assignment.ko.md

### CODE REVIEW

File: 6-consumer/lessons/1-speech-recognition/translations/pi-audio.ko.md [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+
+   만약 다음의 오류가 발생하는 경우:
+
+   ```output
+   OSError: [Errno -9997] Invalid sample rate
+   ```
+
+   44100 또는 16000으로 `rate` 값을 변경합니다.
+
+> 💁 이 코드는 [code-record/pi](../code-record/pi) 폴더에서 찾을 수 있습니다..
```
Comment (WZNT-KimHyeJin) [inline 6-consumer/lessons/1-speech-recognition/translations/pi-audio.ko.md:215]: Can you remove the last dot?

**Review by WZNT-KimHyeJin — COMMENTED** [review]
Except that one, I think you did cool!!!
😝😝😝😝😝

**Comment by WZNT-KimHyeJin** [comment]
@jimbobbennett She has completed the translation of lesson 6's  language-understanding lecture

File: 6-consumer/lessons/1-speech-recognition/translations/pi-audio.ko.md [hunk 2]
```diff
@@ -0,0 +1,217 @@
+# 오디오 캡처 - Raspberry Pi
+
+이 단원에서는 Raspberry Pi로 오디오를 캡처하는 코드를 작성합니다. 오디오 캡처는 버튼으로 제어됩니다.
+
+## 하드웨어
+
+The Raspberry Pi needs a button to control the audio capture.
```
Comment (jimbobbennett) [inline 6-consumer/lessons/1-speech-recognition/translations/pi-audio.ko.md:7]: Should this be translated?

Comment (jhYun505) [reply 6-consumer/lessons/1-speech-recognition/translations/pi-audio.ko.md]: This too. I fixed it.

**Review by jimbobbennett — APPROVED** [review]

**Pull request merged**